ckanext/gla/search.py

```diff
@@ -47,8 +48,10 @@ def add_quality_to_search(search_params):
             #
             # Note also that the text field contains a large amount of
             # metadata copied from other fields in a stemmed form.
+            #
+            #,"qf":"name^4 title^4 tags^2 groups^2 text" # CKAN Defaults
 
-            # "qf":"name^4 title^4 tags^2 groups^2 text"
+            ,"qf":"title^4 notes" # limit matching of text queries to agreed fields
```

Comment:
include search_description here when we merge with highlighting PR.

Highlighting PR specifies:

> "hl.fl": "title,notes,search_description,organization"